cmake/package_settings.cmake

```diff
@@ -43,13 +43,8 @@ set(OE_LIBDIR
     CACHE INTERNAL "Library collector")
 
 # Make directories for build systems (NMake) that don't automatically make them.
-file(
```

Comment:
This is not related with my PR. But I wasn't able to commit it due to format issue.
Running `./scripts/format-cmake` made this change.